cmd/root/root.go

```diff
@@ -77,6 +96,18 @@ func GetCommand() *cobra.Command {
 	return cmd
 }
 
+// defaultPidFilePath returns the default path for the PID file. Notably modern OS X variants
+// do not have /var/run/
+func defaultPidFilePath() string {
+	// If we're on OS X or Windows, use the user's home directory
+	// Otherwise, use /run
+	path := "/run/pget.pid"
+	if runtime.GOOS == "darwin" || runtime.GOOS == "windows" {
+		path = os.Getenv("HOME") + "/.pget.pid"
```

Comment:
https://pkg.go.dev/os#UserHomeDir is slightly nicer I think, although you have to check for err. Or maybe even os.UserCacheDir?